services/src/main/scala/cromwell/services/womtool/WomtoolServiceMessages.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell.services.womtool
+
+import cromwell.core.WorkflowSourceFilesCollection
+import cromwell.services.ServiceRegistryActor.ServiceRegistryMessage
+import spray.json.DefaultJsonProtocol
+
+object WomtoolServiceMessages {
+
+  final val WomtoolServiceName = "Womtool"
+
+  sealed trait WomtoolServiceMessage extends ServiceRegistryMessage {
+    override def serviceName: String = WomtoolServiceName
+  }
+
+  case class DescribeRequest(filesCollection: WorkflowSourceFilesCollection) extends WomtoolServiceMessage
+
+  sealed trait DescribeResult extends WomtoolServiceMessage
+  case class DescribeSuccess(description: WorkflowDescription) extends DescribeResult
+  case class DescribeFailure(reason: String) extends DescribeResult
+
+  case class WorkflowDescription(valid: Boolean, errors: List[String])
```

Comment:
I see you state
>"Otherwise, 'errors' will be empty or unspecified"

in the spec PR but I think having the same set of keys every time (with an empty array) will make life on the frontend easier because one does not have to remember that some fields (but not all!) require a null-check before they're used

Reply:
Agreed.